helm/cluster-operator-chart/templates/configmap.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
 data:
   config.yml: |
     server:
+      enable:
+        debug:
+          server: true
```

Comment:
At least I'd like to have this enabled in all services for three reasons:

* When service gradually builds up memory usage, redeploy would waste time because you cannot immediately get reliable info from heap dump if memory leak is slow (matter of several hours). Debugging during on-call or when working on postmortem would then be quite difficult, I think.
* I don't see this as a performance problem because profiling is effective only on-demand (and depending what you do, you can define the period of time for e.g. CPU profiling).
* This is bound to PODs localhost interface so this is not exposed even inside k8s cluster and always requires `kubectl port-forward` for use. Therefore I don't see any security implications in this.

What do others think?

Reply:
The way you describe it sounds good to me. Don't have so much security knowledge, but when this requires `kubectl port-foreward` sounds good to me.

With `port-foreward` you can do way scarier things than debug anyways.

Reply:
Ping @giantswarm/sig-security

Reply:
lgtm even from security side